lib/fluent/plugin/out_detect_exceptions.rb

```diff
@@ -84,17 +86,19 @@ def emit(tag, es, chain)
 
     def process_record(tag, time_sec, record)
       synchronize do
-        unless @accumulators.key?(tag)
+        accumulator_key = [tag]
```

Comment:
I'd name it for its logical content, i.e., `log_id`, rather its intended purpose (i.e., `accumulator_key`).